### ISSUE TITLE
Any chances that the arrows could be removed from EM-287

### ISSUE THREAD

**Issue by giuseb** [giuseb/histologyguide#27]
[here](https://histologyguide.com/EM-view/EM-287-synapses/06-photo-1.html?x=3789&y=3680&z=25.0&f=1)

**Comment by Brelje** [comment]
I also had issues with this slide. Six months ago, I recolored it to match the newer nervous tissue slides.
Compare:
https://histologyguide.com/EM-view/EM-287-synapses/06-photo-1.html
To:
http://160.94.203.145/EM-view/EM-287-dendritic-spine/06-photo-1.html
The axon terminals (orange) that form synapses with the dendritic spine (green) are now a different color from the other axons and axon terminals (yellow). This renders the arrows unnecessary, especially since a second page has been added to explain the structure of a synapse in greater detail. I like the new version MUCH better.

**Comment by giuseb** [comment]
Yes, the new version is very good.
Another aspect that you may want to consider is that [both symmetrical (inhibitory, red waypoint) and asymmetrical (excitatory, green waypoint) synapses can be seen here](http://160.94.203.145/EM-view/EM-287-dendritic-spine/06-photo-1.html?x=3731&y=5414&z=25.00&page=2&b=b-3-1-0-4763-4685-1975-Click%2520to%2520Show%2520View-1-1-5111-6725-2500-Click%2520to%2520Show%2520View-1-1-2955-4005-2500-Click%2520to%2520Show%2520View), characterized not only by the shape of presynaptic vesicles, but also by the thickness of the post-synaptic density.

**Comment by Brelje** [comment]
Good point.  I recolored the image, changing the colors of the synaptic vesicles:  asymmetric (purple), symmetric (blue), and those in other axon terminals (dark yellow).  [Page 2](http://160.94.203.145/EM-view/EM-287-dendritic-spine/06-photo-1.html?page=2&b=b-3-1-0-4763-4685-1975-Click%2520to%2520Show%2520View-1-1-2955-4005-2500-Click%2520to%2520Show%2520View-1-1-5111-6725-2500-Click%2520to%2520Show%2520View) was rewritten to only describe the three parts of a synapse, and [page 3](http://160.94.203.145/EM-view/EM-287-dendritic-spine/06-photo-1.html?page=3&b=b-3-1-0-4763-4685-1975-Click%2520to%2520Show%2520View-1-1-2955-4005-2500-Click%2520to%2520Show%2520View-1-1-5111-6725-2500-Click%2520to%2520Show%2520View) was added to describe asymmetrical and symmetrical synapses.
